### ISSUE TITLE
The [CLS] token

### ISSUE THREAD

**Issue by hbenyamina** [google-research/bert#1223]
During pre-training, is the [CLS] token ignored when calculating the loss?
And if that is the case, doesn't that mean that my model will predict a random token instead of [CLS]?

**Comment by Abhimanyu08** [comment]
During pre-training the [CLS] representation is fed into binary classifier which predicts whether the sentence pair <A,B> are concurrent sentences or not. 50% of the time <A,B> are actual next sentences in a large corpus and 50% of the time B is a sentence chosen randomly from other document in a large corpus and therefore isn't a valid continuation A. for eg. <Delhi is capital of India. Penguins don't fly>.

**Comment by hbenyamina** [comment]
Is this still valid for RoBERTa ?
What does RoBERTa do with this token?
